python/lsst/ap/pipe/matchApFakes.py

```diff
@@ -164,14 +163,13 @@ def _trimFakeCat(self, fakeCat, image):
         wcs = image.getWcs()
 
         bbox = Box2D(image.getBBox())
-        corners = bbox.getCorners()
-
-        skyCorners = wcs.pixelToSky(corners)
-        region = ConvexPolygon([s.getVector() for s in skyCorners])
 
         def trim(row):
-            coord = SpherePoint(row[self.config.raColName], row[self.config.decColName], radians)
-            return region.contains(coord.getVector())
+            coord = SpherePoint(row[self.config.raColName],
+                                row[self.config.decColName],
+                                radians)
+            cent = wcs.skyToPixel(coord)
```

Comment:
This conversion was already done elsewhere in the code and x and y should be present in row without needing to be recalculated.